packages/core/src/modules/credentials/CredentialsModuleOptions.ts

```diff
@@ -41,7 +39,7 @@ interface NegotiateProposalOptions extends BaseOptions {
 interface OfferCredentialOptions extends BaseOptions {
   credentialRecordId?: string
   connectionId?: string
-  protocolVersion: CredentialProtocolVersion
+  protocolVersion?: CredentialProtocolVersion
```

Comment:
protocolVersion shouldn't be optional for offerCredential, as it's the start of the protocol in this case

Reply:
That change breaks loads of stuff. I suggest leaving this for now

Reply:
What is the reason for this? It is required in the current main branch.

Not keen on merging this change as it's now not clear you're required to pass the protocolVersion

Reply:
I have reverted it back to being mandatory for now.

As stated I would like to allot a PR just for this one Protocol Version field as I find it bewilderingly confusing and I want to fully understand it and do it properly.